public/app/features/canvas/runtime/scene.tsx

```diff
@@ -74,6 +74,7 @@ export class Scene {
   currentLayer?: FrameState;
   isEditingEnabled?: boolean;
   shouldShowAdvancedTypes?: boolean;
+  isQuickDataLinkAccessEnabled?: boolean;
```

Comment:
A nitpick but this would is a bit of a mouthful name wise. Maybe something like `quickDataLinksEnabled` or something like that 😄